Dangerfile

```diff
@@ -0,0 +1,29 @@
+# MessageKit, 2020
+
+# This runs on CI
+mergeable_state = github.pr_json["mergeable_state"]
+
+# Make it more obvious that a PR a draft
+if mergeable_state == "draft"
+  warn("PR is marked as Draft")
+end
+
+# Mainly to encourage writing up some reasoning about the PR, rather than just leaving a title
+if github.pr_body.length < 5
+  fail("Please provide a summary in the Pull Request description")
+end
+
+declared_hashtag = github.pr_title.include?("#trivial")
+hasChangelogEntry = git.modified_files.include?("CHANGELOG.md")
+if !hasChangelogEntry && !declared_hashtag
+  fail("Please include a CHANGELOG entry. \nYou can find it at [CHANGELOG.md](https://github.com/MessageKit/MessageKit/blob/master/CHANGELOG.md).")
+end
+
+# Warn when there is a big PR
+if git.lines_of_code > 1000
+  warn("Big Pull Request - Please consider splitting up your changes into smaller Pull Requests.")
+end
+
+swiftlint.config_file = '.swiftlint.yml'
+swiftlint.lint_files inline_mode:true
+swiftlint.lint_files fail_on_error:true
```

Comment:
@Kaspik - This second line causes swiftlint to run again, I think you have to aggregate the options onto one `swiftlint.lint_files` command.

Reply:
Also, if you look closer at the check phase, we're getting this error (twice):

```
/opt/hostedtoolcache/Ruby/2.6.5/x64/lib/ruby/gems/2.6.0/gems/danger-swiftlint-0.24.1/ext/swiftlint/bin/swiftlint: 1: /opt/hostedtoolcache/Ruby/2.6.5/x64/lib/ruby/gems/2.6.0/gems/danger-swiftlint-0.24.1/ext/swiftlint/bin/swiftlint: Syntax error: "(" unexpected
/opt/hostedtoolcache/Ruby/2.6.5/x64/lib/ruby/gems/2.6.0/gems/danger-swiftlint-0.24.1/ext/swiftlint/bin/swiftlint: 1: /opt/hostedtoolcache/Ruby/2.6.5/x64/lib/ruby/gems/2.6.0/gems/danger-swiftlint-0.24.1/ext/swiftlint/bin/swiftlint: Syntax error: "(" unexpected
```

I tried it on my own CI with verbose turned on and am getting this:

```
Using config file: .swiftlint.yml
Swiftlint will be run from /home/runner/work/<private-repo>/<private-repo>
Swiftlint will exclude the following paths: []
Swiftlint includes the following paths: []
linting with options: {:config=>".swiftlint.yml", :reporter=>"json", :quiet=>true, :pwd=>"/home/runner/work/<private-repo>/<private-repo>", :force_exclude=>true}
Swiftlint will lint the following files: /home/runner/work/<private-repo>/<private-repo>/<private-repo-folder>/TempDangerPoC.swift
/opt/hostedtoolcache/Ruby/2.6.5/x64/lib/ruby/gems/2.6.0/gems/danger-swiftlint-0.24.1/ext/swiftlint/bin/swiftlint: 1: /opt/hostedtoolcache/Ruby/2.6.5/x64/lib/ruby/gems/2.6.0/gems/danger-swiftlint-0.24.1/ext/swiftlint/bin/swiftlint: Syntax error: "(" unexpected
Received from Swiftlint: {}
```
So... not very helpful except that I noticed my `included` paths specified in my .swiftlint.yml are not being honored.

Reply:
Hmmm, I don't think running it twice hurts, but I'll switch it to one-line command.

Yes, there is a bug in SwiftLint, but I didn't have time to look at it yet.